src/jobs/jobs.controller.ts

```diff
@@ -749,11 +737,11 @@ export class JobsController {
     const ability = this.caslAbilityFactory.createForUser(
       request.user as JWTUser,
     );
-    const canCreate =
+    const canRead =
```

Comment:
some test fail with permission problems, i suspect ti could be related to this change here

Reply:
The latest commit fixes the failing tests